### ISSUE TITLE
[Documentation] Wrong documentation for TypeORM

### ISSUE THREAD

**Issue by TomerAmirV** [Romakita/ts-express-decorators#438]
## Informations

Type | Version
---|---
Bug | 4.30.1

****

## Description
A few sentences describing the overall goals of the issue.

In the typeorm documentation, there is an example which says you should use the `AfterInit` interface, but it does not exist

**Comment by Romakita** [comment]
You've right, it's AfterRoutesInit XD

**Comment by TomerAmirV** [comment]
😄 
is `AfterRoutesInit` good?
will this happen before the server is "ready"? since I think that once the routes are loaded, the service should already be provisioned.

**Comment by Romakita** [comment]
The lifecycle is described here :) http://tsed.io/#/docs/services/lifecycle-hooks
If isn't clear, tell me ;)

**Comment by Romakita** [comment]
But the picture src are broken...

**Comment by TomerAmirV** [comment]
maybe it can also be fixed 😄

**Comment by Romakita** [comment]
Yep, the new version of website is almost done. I'll fix that with this new version ;)
